### PR TITLE
doc: Bluetooth RPC Host related documentation updates

### PR DESCRIPTION
**Pull request by ppryga-nordic**
There were updated documentation related to Bluetooth RPC host interface in following parts of SDK:
- nordic-bt-rpc snippet
- BLE RPC subsystem
- ipc_radio
- peripheral_uart

### CODE REVIEW

File: applications/ipc_radio/README.rst [hunk 1]
```diff
@@ -63,8 +63,10 @@ Application
 You can set the supported radio configurations using the following Kconfig options:
 
 * :kconfig:option:`CONFIG_IPC_RADIO_BT` - For the Bluetooth Low Energy serialization.
+* :kconfig:option:`CONFIG_IPC_RADIO_BT_RPC` - For the Bluetooh ost API serialization.
```
Comment (dchat-nordic) [inline applications/ipc_radio/README.rst:66]: The distinction between rpc and hci is described on line 70. Maybe you can move this line (and add one about hci_ipc option) there after line 70?

File: samples/bluetooth/peripheral_uart/README.rst [hunk 2]
```diff
@@ -160,6 +160,17 @@ Building and running
...
+Experimental Bluetooth Low Energy Remote Procedure Call interface
+=================================================================
+
+To build the sample with a Bluetooth Low Energy Remote Procedure Call :ref:`ble_rpc` interface, use the following command:
+
+.. code-block:: console
+
+   west build samples/bluetooth/peripheral_uart -b board_name --sysbuild -S nordic-rpc-host -- -DFILE_SUFFIX=bt_rpc
+
+On the |nRF5340DKnoref|, you can build the sample with Bluetooth Low Energy Remote Procedure Call interface with the :ref:`ipc_radio` image using the same command.
```
Comment (dchat-nordic) [inline samples/bluetooth/peripheral_uart/README.rst:172]: I think this line is unnecessary, because the RPC requires multicore board anyway.

**Review by dchat-nordic — CHANGES_REQUESTED** [review]

File: samples/bluetooth/peripheral_uart/README.rst (hunk 2 above)
Comment (ppryga-nordic) [reply samples/bluetooth/peripheral_uart/README.rst]: Removed

File: applications/ipc_radio/README.rst (hunk 1 above)
Comment (ppryga-nordic) [reply applications/ipc_radio/README.rst]: Changed, added list of choices available in https://github.com/nrfconnect/sdk-nrf/blob/fb4898139020f3f1f7297571ab6b6364d7d179ff/applications/ipc_radio/Kconfig#L22-L41

**Review by dchat-nordic — APPROVED** [review]

File: samples/bluetooth/peripheral_uart/README.rst [hunk 3]
```diff
@@ -160,6 +160,15 @@ Building and running
 
 .. _peripheral_uart_sample_activating_variants:
 
+Experimental Bluetooth Low Energy Remote Procedure Call interface
+=================================================================
+
+To build the sample with a Bluetooth Low Energy Remote Procedure Call :ref:`ble_rpc` interface, use the following command:
```
Comment (peknis) [inline samples/bluetooth/peripheral_uart/README.rst:166]:
```suggestion
To build the sample with a :ref:`ble_rpc` interface, use the following command:
```

File: snippets/nordic-bt-rpc/README.rst [hunk 4]
```diff
@@ -1,11 +1,23 @@
...
+Nordic Bluetooth Low Energy Remote Procedure Call snippet (nordic-bt-rpc)
+#########################################################################
 
 Overview
 ********
 
-This snippet allows users to build Bluetooth applications with Bluetooth stack running on Radio
-core. The communication between an application and Bluetooth stack is handled over the Bluetooth
-RPC subsystem.
+This snippet allows users to build Bluetooth applications with Bluetooth stack running on a core dedicated for radio communication.
```
Comment (peknis) [inline snippets/nordic-bt-rpc/README.rst:9]:
```suggestion
This snippet allows you to build Bluetooth applications with Bluetooth stack running on a core dedicated for radio communication.
```

File: snippets/nordic-bt-rpc/README.rst [hunk 5]
```diff
@@ -1,11 +1,23 @@
...
+For example in the nRF54H20 SOC it is radio core.
```
Comment (peknis) [inline snippets/nordic-bt-rpc/README.rst:10]:
```suggestion
For example in the nRF54H20 SoC it is radio core.
```

File: snippets/nordic-bt-rpc/README.rst [hunk 6]
```diff
@@ -1,11 +1,23 @@
...
+The communication between an application and the Bluetooth stack is handled by the Bluetooth RPC subsystem.
```
Comment (peknis) [inline snippets/nordic-bt-rpc/README.rst:11]:
```suggestion
The communication between an application and the Bluetooth stack is handled by the Bluetooth RPC library.
```

File: snippets/nordic-bt-rpc/README.rst [hunk 7]
```diff
@@ -1,11 +1,23 @@
...
+
+Supported SoCs
+**************
+
+Currently the following SoCs from Nordic Semiconductor are supported for use with the snippet:
```
Comment (peknis) [inline snippets/nordic-bt-rpc/README.rst:16]:
```suggestion
Currently, the following SoCs from Nordic Semiconductor are supported for use with the snippet:
```

File: snippets/nordic-bt-rpc/README.rst [hunk 8]
```diff
@@ -1,11 +1,23 @@
...
+
+* :ref:`nRF5340 <programming_board_names>`
+* :ref:`nRF54H20 <programming_board_names>`
+
+.. note::
+   For the nRF54H20 SOC the snippet modifies memory map to make room for settings storage for radio core.
```
Comment (peknis) [inline snippets/nordic-bt-rpc/README.rst:22]:
```suggestion
   For the nRF54H20 SoC, the snippet modifies the memory map to make room for settings storage of the radio core.
```

File: snippets/nordic-bt-rpc/README.rst [hunk 9]
```diff
@@ -1,11 +1,23 @@
-.. _nordic-rpc-host:
```
Comment (peknis) [inline snippets/nordic-bt-rpc/README.rst:1]: To be on the safe side, keep the both doc labels. If we have earlier linked to this doc using :ref:`nordic-rpc-host` the link will be broken.

**Review by peknis — CHANGES_REQUESTED** [review]

File: snippets/nordic-bt-rpc/README.rst (hunk 9 above)
Comment (dchat-nordic) [reply snippets/nordic-bt-rpc/README.rst:1]: I don't think we linked to this in many places. The ref itself was added 14h ago here: https://github.com/nrfconnect/sdk-nrf/pull/15634/files#diff-36440314ecd7741e2df0b84b51ba008dc0a72e83b56995def3c56979de7e4945

Comment (peknis) [reply snippets/nordic-bt-rpc/README.rst:1]: It does no harm if we keep it there. Up to you.

File: snippets/nordic-bt-rpc/README.rst [hunk 10]
```diff
@@ -1,11 +1,23 @@
...
+For example in the nRF54H20 SOC it is radio core.
+The communication between an application and the Bluetooth stack is handled by the Bluetooth RPC subsystem.
+
+Supported SoCs
+**************
+
+Currently the following SoCs from Nordic Semiconductor are supported for use with the snippet:
+
+* :ref:`nRF5340 <programming_board_names>`
+* :ref:`nRF54H20 <programming_board_names>`
```
Comment (peknis) [inline snippets/nordic-bt-rpc/README.rst:19]: Alternatively, you can link directly to these SoCs as follows:

* :ref:`zephyr:nrf5340dk_nrf5340`
* :ref:`zephyr:nrf54h20dk_nrf54h20`

**Review by peknis — CHANGES_REQUESTED** [review]

**Comment by gmarull** [comment]
please push branches to forks

**Comment by ppryga-nordic** [comment]
> please push branches to forks

I'm sorry! I'll re-open it from my fork.

File: snippets/nordic-bt-rpc/README.rst (hunk 9 above)
Comment (ppryga-nordic) [reply snippets/nordic-bt-rpc/README.rst:1]: No there is no reference to the former link. I'll leave the new one only.

File: samples/bluetooth/peripheral_uart/README.rst (hunk 3 above)
Comment (ppryga-nordic) [reply samples/bluetooth/peripheral_uart/README.rst:166]: Done

File: snippets/nordic-bt-rpc/README.rst (hunk 4 above)
Comment (ppryga-nordic) [reply snippets/nordic-bt-rpc/README.rst:9]: Done

File: snippets/nordic-bt-rpc/README.rst (hunk 6 above)
Comment (ppryga-nordic) [reply snippets/nordic-bt-rpc/README.rst:11]: Done

File: snippets/nordic-bt-rpc/README.rst (hunk 7 above)
Comment (ppryga-nordic) [reply snippets/nordic-bt-rpc/README.rst:16]: Done

**Pull request closed without merging**